vmware/vmware-cloud-director-ui-components
# Make labels of all controls be grayed out when disabled

## PR Checklist

Please check if your PR fulfills the following requirements:


## PR Type

What kind of change does this PR introduce?




## What does this change do?
Apply disabled styles for the select component.
Adding an example for showing the disabled and readOnly functionality.

## What manual testing did you do?

* Go to `/formInputs/example`
* Check the box to disable components
    * Verify that you cannot interact with components and that their label is grayed out.
* Do the same check for `/formCheckbox/example`
* Same check for `/numberWithUnitFormInput/example/number-with-unit-form-input`

* Go to `/formSelect/example/form-select-disabled`
* Verify that the disabled select's label is grayed out and that you cannot interact with it.

## Screenshots (if applicable)
![Screenshot 2022-12-14 at 15 31 17](https://user-images.githubusercontent.com/159775/207834100-1a134696-f7a4-4712-a309-dcd8ec349055.png)

<img width="658" alt="image" src="https://user-images.githubusercontent.com/549331/208193415-27ffa207-8c30-42ba-8bfa-070b9c60b7a4.png">

<img width="669" alt="image" src="https://user-images.githubusercontent.com/549331/208193493-9f0e0088-3d58-4cd1-b869-cf0741e455ce.png">
<img width="943" alt="image" src="https://user-images.githubusercontent.com/549331/208194831-3e683616-a59b-4130-b902-9339ebc9b8d2.png">

## Does this PR introduce a breaking change?




## Other information
